src/d3a/models/market/market_redis_connection.py

```diff
@@ -117,19 +117,20 @@ def _delete_offer_response_channel(self):
     def _accept_offer_response_channel(self):
         return f"{self._accept_offer_channel}/RESPONSE"
 
-    @classmethod
-    def _parse_payload(cls, payload):
+    def _parse_payload(self, payload):
         data_dict = json.loads(payload["data"])
-        return cls.sanitize_parameters(data_dict)
+        return MarketRedisEventSubscriber.sanitize_parameters(data_dict, self.market.now)
```

Comment:
I think it is okay to use `cls` here instead of the class name.

Reply:
Unfortunately it won't work since I had to change `_parse_payload ` from classmethod to instance method

Reply:
I see, because you need the current time, all good then